cmds/mpu.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Phoenix-RTOS loader
+ *
+ * mpu command
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "cmd.h"
+
+#include <hal/armv7m/mpu.h>
```

Comment:
I think that `mpu.h` should be included in `config.h` in hal, then you would use `#include <hal/hal.h>` 
The same case was done in `bitstream` command.

Reply:
That's right. Done.

Reply:
That's right. Done.